src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/PaymentBuilderWidget.tsx

```diff
@@ -136,30 +186,88 @@ const PaymentBuilderWidget: FC<PaymentBuilderWidgetProps> = ({ action }) => {
-            text={formatText({
-              id: 'expenditure.fundingStage.info',
-            })}
-            content={
-              <Button
-                className="w-full"
-                onClick={showFundingModal}
+          <>
+            {isFundingMotion && !isFundingMotionFailed && !isFundingClaimed ? (
+              <div className="flex flex-col gap-2">
+                <RequestBox
+                  title={formatText({ id: 'expenditure.fundingRequest.title' })}
+                  motions={fundingMotions}
+                />
+                <MotionBox
+                  transactionId={
+                    selectedTransaction ||
+                    motions?.items[0]?.transactionHash ||
+                    ''
+                  }
+                />
+              </div>
+            ) : (
+              <StepDetailsBlock
                 text={formatText({
-                  id: 'expenditure.fundingStage.button',
+                  id: 'expenditure.fundingStage.info',
                 })}
-                loading={expectedStepKey === ExpenditureStep.Release}
+                content={
+                  <>
+                    {fundingMotions && fundingMotions.length > 0 && (
+                      <RequestBox
+                        withoutPadding
+                        title={formatText({
+                          id: 'expenditure.fundingRequest.title',
+                        })}
+                        motions={fundingMotions}
+                      />
+                    )}
+                    <Button
+                      className="w-full"
+                      onClick={showFundingModal}
+                      text={formatText({
+                        id: 'expenditure.fundingStage.button',
+                      })}
+                      loading={expectedStepKey === ExpenditureStep.Release}
+                    />
+                  </>
+                }
               />
-            }
-          />
+            )}
+          </>
         ) : (
-          // @todo: please update this element when other decisions methods for funding will be implemented
           <>
-            {fundingActionsItems?.[0] && (
+            {isFundingMotion ? (
+              <div className="flex flex-col gap-2">
+                <RequestBox
+                  title={formatText({ id: 'expenditure.fundingRequest.title' })}
+                  motions={fundingMotions}
+                />
+                <MotionBox
+                  transactionId={
+                    selectedTransaction ||
+                    motions?.items[0]?.transactionHash ||
+                    ''
+                  }
+                />
+              </div>
+            ) : undefined}
+            {fundingActionsItems?.length === 1 && (
               <FinalizeWithPermissionsInfo
-                userAdddress={fundingActionsItems[0].initiatorAddress}
+                userAdddress={fundingActionsItems[0]?.initiatorAddress}
               />
```

Comment:
Why are we only showing the funding info if there's exactly one funding motion? Technically the UI only allows one for now, but if you create another one on chain, they all disappear.

Reply:
This chunk of code is used for permissions, it's going to be handled in other issue.

Reply:
Sure, but this check feels unnecessarily strict. You can create multiple funding actions with permissions which would hide everything. I think showing at least the first action, while not perfect, is preferable to hiding them all.